backend/inspirehep/indexer/tasks.py

```diff
@@ -72,8 +85,10 @@ def index_record(self, uuid, record_version=None, force_delete=None):
         list(dict): Statistics from processing references.
     """
     LOGGER.debug("Indexing record", uuid=str(uuid), version=record_version)
-    record = get_record(uuid, record_version)
-
+    record = InspireRecord.get_record(
+        uuid, with_deleted=True, record_version=record_version
+    )
+    LOGGER.debug(f"INDEXING {record['control_number']} RECORD")
```

Comment:
```suggestion
    LOGGER.debug(f"Indexing {record['control_number']} record")
```

Reply:
Ah I will remove this, it was for my debugging yesterday. Not needed anymore.